inc/classes/optimization/CSS/class-minify.php

```diff
@@ -119,8 +119,11 @@ private function replace_url( $url ) {
 			return false;
 		}
 
+		// This filter is documented in /inc/classes/optimization/class-abstract-optimization.php.
+		$url = apply_filters( 'rocket_asset_url', $url, $this->get_zones() );
```

Comment:
Why are we adding this filter? Is it a feature request? _curious_

Reply:
I'm using it for 2 things:
- Reduce coupling we had before with the CDN part of the plugin
- When CDN is enabled, I convert the CDN URL back to the origin URL, to make sure the file has the correct path when we save it in the `min` folder. Else we could get unexpected paths when the CND URL itself has a path.